EasyPost.Tests/Fixture.cs

```diff
@@ -706,5 +709,23 @@ public static string Event
                 };
             }
         }
+        public static byte[] EventBody
+        {
+            get
+            {
+                const string relativePath = "eventBody.json";
+                string fullPath = Path.Combine(TestUtils.GetSourceFileDirectory(), relativePath);
+
+                try
+                {
+                    string jsonString = File.ReadAllText(fullPath).Replace("\n", "").Replace("\r", ""); // remove potential newline at end of JSON file, it works, don't touch it :)
```

Comment:
Issue: Just read the first line of the file, it's much less brittle to do that than this double replace. Go and Java read the first line, you can use those as an example.

Reply:
Not an issue, but addressed.

Reply:
An issue is something that must be fixed before the reviewer will approve. Please re-review the conventional comments internal doc.